### PR TITLE
Update serving docker file

### PR DESCRIPTION
**Pull request by frankfliu**
## Channel for questions ##
Before or while filing an issue please feel free to join our [Slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/zt-ar91gjkz-qbXhr1l~LFGEIEeGBibT7w) to get in touch with development team, ask questions, find out what's cooking and more!

## Description ##
(Brief description of what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, Java doc)
- [ ] Feature2, tests, (and when applicable, Java doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: serving/docker/Dockerfile [hunk 1]
```diff
@@ -23,6 +23,12 @@ EXPOSE 8080
 
 WORKDIR /opt/djl
 ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+ENV OMP_NUM_THREADS=1
+ENV TF_NUM_INTEROP_THREADS=1
+ENV TF_NUM_INTRAOP_THREADS=1
+ENV TF_CPP_MIN_LOG_LEVEL=1
+ENV MXNET_ENGINE_TYPE=NaiveEngine
+ENV JAVA_OPTS="-Dai.djl.disable_close_resource_on_finalize=true"
```
Comment (stu1130) [inline serving/docker/Dockerfile:31]: -Dai.djl.pytorch.num_interop_threads=1 ?

Comment (frankfliu) [reply serving/docker/Dockerfile:31]: Do we need ai.djl.pytorch.num_interop_threads? I thought it has no impact

Comment (stu1130) [reply serving/docker/Dockerfile:31]: From what we observed on resnet18 yes. But it should have impact when operators in the model can be run concurrently

Comment (stu1130) [reply serving/docker/Dockerfile:31]: We can change it later if we find one

**Review by stu1130 — APPROVED** [review]
